### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by calvnce**

### CODE REVIEW

File: dist/js/script.js [hunk 1]
```diff
@@ -3,3 +3,49 @@
...
+   */
+  function classItem(className) {
+    return document.querySelector(className);
+  }
+
+  // Variables
+  let isOpen = false;
+  const menu = classItem('.mobile-menu-button');
+  const menuItem = classItem('.mobile-menu-button-item');
+  const nav = classItem('.desktop-menu');
```
Comment (smcommits) [inline dist/js/script.js:38]: - [ ]  **Please make sure that the user click (or tap) any option of the mobile menu, the mobile menu disappears. Also, make sure that the links are working**

This is the default behavior in mobile menus. As soon as a link is clicked, the menu should close and the user should be taken on the respective link. Please implement this.

Comment (smcommits) [inline dist/js/script.js:38]: - [ ] **Please make sure that the menu matches the original design from Figma.**

The menu in original design spans over the entire height of the viewport. Please modify your menu to match the original design 

Cloned Design|Orginal Design
:------------------:|:-------------------:
image|image

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @calvnce, 

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. 
### To make this project even better, there are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️
_Please check the inline comments below._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by smcommits** [comment]
@calvnce A mistake has been made and this review has been marked as "Approved". I am extremely about this. Please don't submit the project as the project will be marked as "Requested Changes" as soon as possible. Again, sorry for the accident. Please feel free to leave a comment if you need any clarification.

**Comment by calvnce** [comment]
This `Pull request` updates the required changes on the mobile menu interactivity

File: dist/js/script.js [hunk 2]
```diff
@@ -3,3 +3,82 @@
...
+   */
+  function classItems(className) {
+    return document.querySelectorAll(className);
+  }
+
+  // Variables
+  let isOpen = false;
+  const menu = classItem('.mobile-menu-button');
+  const menuItem = classItem('.mobile-menu-button-item');
+  const nav = classItem('.desktop-menu');
```
Comment (Temmarie) [inline dist/js/script.js:38]: - [ ] As requested by the previous code reviewer, make sure the menu and other parts of the page matches the original design. 
_See images below._

**Original** | **Your clone**
  ----------- | -----------
  image | image
image | image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @calvnce     ! :wave: :wave:

#### You have done really well in implementing the previously required changes.

 However, there are changes required for you to make in order for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:

- - -

### **Required Changes** ♻️

- [ ] Kindly add a descriptive Pull Request Message with a descriptive title as well. It's essential that your repository is professional, as this is one of the things that potential colleagues will look at when recruiting team members. You currently have a comment underneath the first review. Please add the PR message at the top where the description of what you've done in this milestone should be.

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 1 more review as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Pull request merged**